WalletWasabi.Fluent/Behaviors/ApplicationUtils.cs

```diff
@@ -0,0 +1,18 @@
+using System.Reactive.Linq;
+using System.Reactive.Threading.Tasks;
+using Avalonia;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public static class ApplicationUtils
```

Comment:
This should be moved from Behaviors folder

Reply:
Also aren't there more places where this can be used and unified across code base?

Reply:
Moved.
We can inspect if this can be used in other places in the codebase, but I think this could go into a different PR. What do you think?